### ISSUE TITLE
Taipy cloud UI has character limit on env vars and it's too short

### ISSUE THREAD

**Issue by tommysteryy** [Avaiga/taipy#677]
**Description**
When trying to deploy my `taipy` app through [taipy cloud](https://cloud.taipy.io/cloud/), I want to add `AWS_SECRET_ACCESS_KEY` into the app's environment variables, to access some thing I need in my app. However, when typing it in, there seems to be a hard character limit and I cannot type the full name I need.

This screenshot is taken when I'm hitting `Y` on my keyboard, trying to complete it, but it will not let me.
<img width="728" alt="Screenshot 2024-01-14 at 11 18 03 PM" src="https://github.com/Avaiga/taipy/assets/55601970/53b6b22d-d420-46e2-b270-303ca256ae7b">

**How to reproduce**

1. Go to https://cloud.taipy.io/cloud/
2. Do everything you need to start a machine
3. When configuring your app, in the "**Environment variables**" tab:
    <img width="428" alt="Screenshot 2024-01-14 at 11 22 21 PM" src="https://github.com/Avaiga/taipy/assets/55601970/c18d3ef3-294c-4f5d-b449-161b280da5e1">
4. Try to type in `AWS_SECRET_ACCESS_KEY` as the "SECRET NAME"

**Expected behavior**
It should let me use this secret name in the deployment.

**Screenshots**
Same screenshots as above.

**Runtime environment**
I'm on Microsoft Edge.

**Acceptance Criteria**
- [ ] Ensure new code is unit tested, and check code coverage is at least 90%
- [ ] Create related issue in taipy-doc for documentation and Release Notes if relevant

**Comment by florian-vuillemot** [comment]
Indeed, there is a limit to the length but we'll be able to increase it. Taipy-cloud is currently undergoing a major migration. We'll be taking care of that shortly after.

**Comment by tommysteryy** [comment]
Thanks both for responding to my issue! I'm just wondering, about when is the taipy-cloud migration planned for? When do you think I would be able to expect this fix?
